### DISCUSSION TITLE
Where to look for Socks5 proxy with UDP supported

### DISCUSSION THREAD

**Discussion by Jackychans** [xjasonlyu/tun2socks#188] [Q&A] [answered]
Hello, 
Sorry if this is wrong place to ask this question.
I'm looking all over places but most of proxy providers always provide Socks5 protocol  without UDP supported. So your real IP might be revealed or N/A in Webrtc field

Can anyone advise ?

**Comment by xjasonlyu** [comment, marked as answer]
There're plenty of SOCKS5 server implementations with UDP support:

- [Dante](https://www.inet.no/dante/) _(recommend)_
- [clash](https://github.com/Dreamacro/clash)
- [gost](https://v2.gost.run/en/socks/)
- [v2ray](https://github.com/v2fly/v2ray-core)

**Reply by Jackychans** [reply]
Wow, great anwser.

I'm consuming lots of different proxy providers (Socks5), but very very few of them can enable UDP port open. Most of them UDP ports are closed so webrtc fields show N/A when checking IP leaks.  As of now, I'm usually using tune2socks clients to connect proxy

Do you have some recommendation of good proxy providers who can enable this feature ?

**Reply by xjasonlyu** [reply]
Sorry, no. But you can host that on your own in AWS.

**Reply by Jackychans** [reply]
I'd like to contact you but your telegram is not existed :(

**Reply by xjasonlyu** [reply]
Oh yes, I hid my username due to too spams.

**Reply by Jackychans** [reply]
How can I contact ? I have some interesting points, need to talk
